zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -105,6 +106,7 @@ fn empty_v5_round_trip() {
     zebra_test::init();
 
     let tx = Transaction::V5 {
+        network_upgrade: Nu5,
```

Comment:
Please turn this into a named constant, rather than hard-coding NU5 multiple times.

That will make it easier to update these tests for the next network upgrade.